go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/containerd/cgroups v1.0.2
 	github.com/containerd/containerd v1.5.7
 	github.com/containerd/cri-containerd v1.19.0
+	github.com/containerd/nri v0.1.0
```

Comment:
Are there any plans to move the NRI code out of containerd to make it more independent from the project?

Reply:
If our proposal to extend the scope gets eventually accepted then we'd like to move it somewhere to more neutral ground.